src/blocks/pagination/Pagination.tsx

```diff
@@ -0,0 +1,106 @@
+import Pagination from 'rc-pagination';
+import styled from 'styled-components';
+import 'rc-pagination/assets/index.css';
+import { FC } from 'react';
+import { CaretLeft, CaretRight } from 'blocks/icons';
+import { getTextVariantStyles } from 'blocks/Blocks.utils';
+import { PaginationProps } from './Pagination.types';
+import enUS from 'rc-pagination/lib/locale/en_US';
+
+const StyledPagination = styled(Pagination)<{ disabled?: boolean }>`
```

Comment:
<img width="364" alt="Screenshot 2024-08-26 at 12 45 55 PM" src="https://github.com/user-attachments/assets/5ed0fc2f-0084-42b2-aadf-587d5fae9204">

Disabled mode UI is also broken.